includes/wikia/models/PromoImage.class.php

```diff
@@ -2,10 +2,7 @@
 
 /**
  * Helper class for handling images used for promotion
- *
- *
  */
-
 class PromoImage extends WikiaObject {
 	const __MAIN_IMAGE_BASE_NAME = 'Wikia-Visualization-Main';
 	const __ADDITIONAL_IMAGES_BASE_NAME = 'Wikia-Visualization-Add';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this constant "__ADDITIONAL_IMAGES_BASE_NAME" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)